packages/create-playkit-plugin/createPlaykitPlugin.js

```diff
@@ -129,19 +133,68 @@ if (program.info) {
         .then(console.log);
 }
 
-if (typeof projectName === 'undefined') {
-    console.error('Please specify the project directory:');
-    console.log(
-        `  ${chalk.cyan(program.name())} ${chalk.green('<project-directory>')}`
-    );
-    console.log();
-    console.log('For example:');
-    console.log(`  ${chalk.cyan(program.name())} ${chalk.green('my-react-app')}`);
-    console.log();
-    console.log(
-        `Run ${chalk.cyan(`${program.name()} --help`)} to see all options.`
-    );
-    process.exit(1);
+async function askProjectOptions() {
+    const validation = (pattern = /^[a-zA-Z0-9-]*$/g) => input => {
+        if (!input) {
+            return "This field should not be empty.";
+        }
+
+        if (!pattern.test(input)) {
+            return "It is not valid! Please try again.";
+        }
+
+        return true;
+    };
+
+    const askName = {
+        type: "text",
+        name: "name",
+        message: `Please specify the project name:`,
+        validate: validation(),
+    };
+
+    const askNpmName = name => ({
+        type: "text",
+        name: "npmName",
+        message: "Please specify the plugin NPM name ( default to @playkit-js/{name}-plugin )",
+        validate: validation(/^(@[a-z-]+\/[a-z-]+|[a-z-]+)*$/g),
+        initial: `@playkit-js/${name}-plugin`,
+    })
+
+    const askGithubRepo = name => ({
+        type: "text",
+        name: "githubRepo",
+        message: `Please specify the plugin GitHub repository ( default to kaltura/playkit-js-{name} )`,
```

Comment:
no need to write the `default to...` part, it is the initial value. 

same goes for other questions here